src/trajectory.jl

```diff
@@ -227,8 +227,8 @@ samplecand(rng, τ::StaticTrajectory{EndPointTS}, h, z) = step(τ.integrator, h,
 
 ### Multinomial sampling from trajecory
 
-function randcat(rng::AbstractRNG, p_unorm::AbstractVector{T}) where {T}
-    p = p_unorm ./ sum(p_unorm)
+function randcat(rng::AbstractRNG, up::AbstractVector{T}) where {T}
```

Comment:
Maybe rename `up` to `unnormlised_prob` for readability? `up` is really hard to parse...

Reply:
Sounds good.